README.md

```diff
@@ -174,6 +174,19 @@ curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyy
 curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/profile
 ```
 
+## Development Setup ( Docker )
```

Comment:
Am I correct in assuming that the motivation to setup `vets-api` in Docker locally would be for those who don't want to fuss with `vets-api` at all, such as front-end developers? (sorry, I'm no Docker expert).  If so, we should add a link to our existing [Developer Setup](https://github.com/department-of-veterans-affairs/vets-api/blob/cdafd7e052b09e78f4e07518ea7eecbd3cdff1bf/README.md#developer-setup) section that reads, "Use Docker for vets-api" that anchor links to your "Development Setup (Docker)" section.